Quizzler-iOS13/Controller/ViewController.swift

```diff
@@ -12,11 +12,14 @@ class ViewController: UIViewController {
     
     @IBOutlet weak var questionLabel: UILabel!
     @IBOutlet weak var progressBar: UIProgressView!
-    @IBOutlet weak var trueButton: UIButton!
-    @IBOutlet weak var falseButton: UIButton!
     @IBOutlet weak var scoreLabel: UILabel!
+    @IBOutlet weak var choice1: UIButton!
+    @IBOutlet weak var choice2: UIButton!
+    @IBOutlet weak var choice3: UIButton!
+    //@IBOutlet var choiceButtons: [UIButton]! //could have been better with a list
```

Comment:
@IBOutlet can work with array of items, why is it commented?

Reply:
At first, I wanted to use an array of UIButtons to separate each choice but I couldn't figure out how to get which button was pressed. 
In this approach, I am detecting each button separately.